CONFIGURATION.md

```diff
@@ -19,6 +19,7 @@ Mewpoke get all infos from his configuration's file ```config.yml```
 - `staticDns` use static host as endpoint for discovery
  -- *clustername*: name of the cluster
  -- *host*: hostname
+ -- *bucketpassword*: bucket's password specified in the "Access Control" part of Bucket settings (all buckets within the cluster should have the same password / this feature will disable latency metrics)
```

Comment:
I would put bucketpassword in the 'service' configuration, not in the discovery.